Database/Corrections/cataQuestFixes.lua

```diff
@@ -1214,26 +1221,50 @@ function CataQuestFixes.Load()
         [13708] = { -- Valiant Of Sen'jin
             [questKeys.requiredRaces] = raceKeys.ALL_HORDE,
         },
+        [13712] = { -- To The Rescue
+            [questKeys.extraObjectives] = {{nil,Questie.ICON_TYPE_TALK,l10n("Speak to Kadrak"),0,{{"monster",8582}}}},
+        },
+        [13730] = { -- Playing With Felfire
+            [questKeys.preQuestSingle] = {13803},
+        },
         [13766] = { -- Closure is Only Natural
             [questKeys.objectives] = {{{33767,nil,Questie.ICON_TYPE_EVENT}}},
         },
         [13796] = { -- The Forest Heart
             [questKeys.preQuestSingle] = {13792},
             [questKeys.requiredSourceItems] = {45571,45572},
         },
+        [13798] = { -- Rain of Destruction
+            [questKeys.objectives] = {{{33688}},nil,nil,nil,{{{33945,33195},33195,"Attacking elves slain"}}},
```

Comment:
For this quest you need to add something else. By default Questie will match the objectives the API returns in order with the objectives. That would mean that the `monster` objective would be taken as "Attacking Elves Slain" and the `killCreditObjective` would be for the Ancients.

To fix this order mismatch, at the top of this file you need to add `QuestieCorrections.killCreditObjectiveFirst[13798] = true` (there are other entries already).